Modules/TOF/src/TrendingRate.cxx

```diff
@@ -97,8 +97,13 @@ void TrendingRate::computeTOFRates(TH2F* h, std::vector<int>& bcInt, std::vector
     return;
   }
 
-  if (mActiveChannels) {
-    mNoiseRatePerChannel = (hback->GetMean() - 0.5) / orbit_lenght * h->GetNbinsX() / mActiveChannels;
+  if (mActiveChannels > 0.f) {
+    if (orbit_lenght > 0.f) {
+      mNoiseRatePerChannel = (hback->GetMean() - 0.5) / orbit_lenght * h->GetNbinsX() / mActiveChannels;
+    } else {
+      ILOG(Warning, Support) << "Unphysical orbit_lenght " << orbit_lenght << ", setting noise rate per channel to 0" << ENDM;
+      mNoiseRatePerChannel = 0.f;
+    }
```

Comment:
Line 100-106 not needed.
Check on mActiveChannels was already done in previous version but as integer (it is an integer)
Check on orbit_length not needed since it is a constant
I would remove all this part from the commit